src/Scan.js

```diff
@@ -485,8 +485,15 @@ class Scan extends React.Component {
     }, this.setFocusInput);
   }
 
-  handleTextError(error) {
-    throw error;
+  handleTextError() {
```

Comment:
Why is the signature of this method changing given [there is no change to how it is called](https://github.com/folio-org/ui-checkin/blob/372cb3982c84c9940b971f71d02471494ef43bf9/src/Scan.js#L476)? It doesn't feel like we should throw that information away.

In addition to showing the localized "unhandled error" message (l10n is good!) `message` could/should we also store the root error, e.g. sth like 
```
<div>
  <FormattedMessage id="ui-checkin.errorModal.unhandledError" />
  {error}
</div>
```

Reply:
The code was changed.
So now if we got error differ than `barcode error`, we show warning that this is `unhanded case` and error message if it is present.
Am I correctly understood your proposal?

Reply:
Yes, I see the function's signature has changed and it now ignores its argument. My proposal is, given that we get an argument, show the argument.

Reply:
Side note: please don't force-push commits to an existing branch; this destroys the history and makes conversation threads like this one difficult to understand because they now lack context. Instead, just add new commits and then squash when you merge. This preserves the history within the PR while also keeping an easy-to-follow linear history on the main branch. The best of both worlds!